### PR TITLE
Make the number of boss I/O threads configurable

### LINKED ISSUE #348: Make the number of boss I/O threads configurable
In the Server.java file I saw "bossGroup = new EpollEventLoopGroup(1, bossThreadFactory);".
I wonder when the concurrency level is high, is only one thread enough for the server acceptor?
why not make it configurable?

### PR DESCRIPTION
**Pull request by imasahiro**
Fixes #348

### CODE REVIEW

File: core/src/main/java/com/linecorp/armeria/server/ServerConfig.java [hunk 1]
```diff
@@ -267,7 +276,7 @@ public VirtualHost defaultVirtualHost() {
      * Returns the {@link List} of available {@link VirtualHost}s.
      *
      * @return the {@link List} of available {@link VirtualHost}s where its last {@link VirtualHost} is
-     *         {@link #defaultVirtualHost()}
+     * {@link #defaultVirtualHost()}
```
Comment (trustin) [inline core/src/main/java/com/linecorp/armeria/server/ServerConfig.java]: Revert?

**Comment by trustin** [comment]
LGTM except one nit

File: core/src/main/java/com/linecorp/armeria/server/ServerConfig.java (hunk 1 above)
Comment (imasahiro) [inline core/src/main/java/com/linecorp/armeria/server/ServerConfig.java]: Fixed

**Comment by trustin** [comment]
Thanks, @imasahiro !

**Pull request merged**